src/Components/CreateImageWizard/formComponents/Packages.js

```diff
@@ -159,7 +159,7 @@ const Packages = ({ getAllPackages, isSuccess }) => {
       }
       setChosenPackages(newChosenPackages);
     }
-  }, []);
+  }, [selectedPackages]);
```

Comment:
Use this pattern instead:
https://react.dev/learn/you-might-not-need-an-effect#adjusting-some-state-when-a-prop-changes

Reply:
as we discussed in the group channel, we agreed to add those dependencies
and in the future maybe we will refector the code and remove the useEffect hook were ever we can